.github/release-drafter-config.yml

```diff
@@ -1,31 +1,49 @@
-name-template: 'Version $NEXT_PATCH_VERSION'
-tag-template: 'v$NEXT_PATCH_VERSION'
+name-template: '$NEXT_MAJOR_VERSION'
+tag-template: 'v$NEXT_MAJOR_VERSION'
+autolabeler:
+  - label: 'maintenance'
+    files:
+      - '*.md'
+      - '.github/*'
+  - label: 'bug'
+    branch:
+      - '/bug-.+'
+  - label: 'maintenance'
+    branch:
+      - '/maintenance-.+'
+  - label: 'feature'
+    branch:
+      - '/feature-.+'
 categories:
-  - title: 'Features'
+  - title: 'Breaking Changes'
+    labels:
+      - 'breakingchange'
+
+  - title: '🧪 Experimental Features'
+    labels:
+      - 'experimental'
+  - title: '🚀 New Features'
     labels:
       - 'feature'
       - 'enhancement'
-      - 'c:feature'
-      - 'c:enhancement'
-  - title: 'Bug Fixes'
+  - title: '🐛 Bug Fixes'
     labels:
       - 'fix'
       - 'bugfix'
       - 'bug'
-      - 'c:bug'
-  - title: 'Breaking Changes'
-    labels:
-      - 'pr:break'
-  - title: 'Documentation'
-    labels:
-      - 'x:docs'
-  - title: 'Maintenance'
-    label: 'chore'
+      - 'BUG'
+  - title: '🧰 Maintenance'
+    label: 'maintenance'
 change-template: '- $TITLE (#$NUMBER)'
 exclude-labels:
   - 'skip-changelog'
-  - 'x:build'
```

Comment:
If we don't skip `build` PRs, at least let's give it its own section so it would be easy to browse through them and decide what should be mentioned in RN and what should be skipped

Reply:
@oshadmi Do we know what all these labels are for? By all means, feel free to change as makes sense for your project - but I'd love to know what they all do, or if they're historic + inertia based!

Reply:
There are several [labels](https://github.com/RediSearch/RediSearch/labels)
Used for triaging/planning/prioritization, classification, etc.
Some are historic, e.g., `1.4 only`, but most of them seem relevant.

Reply:
@oshadmi want to change these based on how your team uses them? Unfortunately I can't know what's relevant in this project.